src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -616,8 +618,29 @@ public static String getReportFilename(IReport selectedReport) {
      * @throws FileNotFoundException
      */
     public static String createReport(IReport report, IInternalContest contest, IInternalController controller, boolean printHeaderAndFooter) throws FileNotFoundException {
+        return createReport(Constants.REPORT_DIRECTORY_NAME, report, contest, controller, printHeaderAndFooter);
+    }
+    
+    /**
+     * Create/Write a report to file in outputDirectoryName, creates outputDirectoryName if dir doesnot exist. 
+     * 
+     * 
+     * @param outputDirectoryName output directory name
+     * @param report
+     *            IReport to write
+     * @param contest
+     * @param controller
+     * @return name of created report file.
+     * @throws FileNotFoundException
+     */
+    public static String createReport(String outputDirectoryName, IReport report, IInternalContest contest, IInternalController controller, boolean printHeaderAndFooter) throws FileNotFoundException {
```

Comment:
The Javadoc header for this method is incomplete (it's missing a description of the `printHeaderAndFooter` parameter).